docs/eventing/sink-binding.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "Sink Binding"
+weight: 60
+type: "docs"
+---
+
+![version](https://img.shields.io/badge/API_Version-v1-red?style=flat-square)
+
+The `SinkBinding` custom object supports decoupling event production from delivery addressing.
+
+You can use sink binding to connect event producers, such as an event source, to an addressable Kubernetes object that can receive events, also known as an event sink.
+
+Sink binding injects environment variables into the `PodTemplateSpec` of the event sink, so that the application code does not need to interact directly with the Kubernetes API to locate the event destination.
+
+# Prerequisites
+
+Before you can use sink binding, you must ensure that:
+- Knative Serving and Eventing are installed on your cluster.
```

Comment:
Serving is not a dependency of sink binding, just maybe of this demo code. The only dependency is eventing.

Reply:
Yeah it's just for the sake of a tutorial if we use a ksvc for a sink.
I'll do some changes to make it optional and call out better that it's for the tutorial rather than sinkbinding on its own.